roles/azuracast/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+# A full-featured online radio station suite.
+# Works on Ubuntu 18.04, Debian 9, 10. Uses docker
+
+# azuracast_install: False
+# azuracast_enabled: False
+
+# All above are set in: github.com/iiab/iiab/blob/master/vars/default_vars.yml
+# If nec, change them by editing /etc/iiab/local_vars.yml prior to installing!
+
+azuracast_http_port: 8080
```

Comment:
Is 8080 really a good idea?  Several IIAB countries/groups use Calibre on port 8080 :/

(@m-anish can you possibly make another suggestions for AzuraCast?)

And document both these AzuraCast ports in https://github.com/iiab/iiab/wiki/IIAB-Networking#list-of-ports--services ?

PS i'm torn as to what to do w.r.t. local_vars_medium.yml and local_vars_big.yml if this does not yet run on Raspberry Pi, but let's discuss various recommendations here over Live Chat if poss.

Reply:
it can be any port we want. pick one?

Reply:
Please also take the initiative to propose an AzuraCast port, that is different from 8080.

Then post these 2 ports to: https://github.com/iiab/iiab/wiki/IIAB-Networking#list-of-ports--services

Reply:
We're now using 10080/10443 for http/https and generally 10000-10100 for azuracast connections.